### PR TITLE
Prevent flash between ModalOverylay and TourOverlay

### PR DESCRIPTION
**Pull request by microbit-robert**
TourOverlay seems to do the right thing with undefined values passed in.

Steps to reproduce flash before fix:

- Record your first sample and see tour popup
- Go forward and back between the first and second tour step and see the modal overlay flash

### CODE REVIEW

File: src/pages/Tour.tsx [hunk 1]
```diff
@@ -84,18 +83,14 @@ const Tour = () => {
...
-          padding={spotlightPadding}
-          paddingTop={step.spotlightStyle?.paddingTop}
-          paddingBottom={step.spotlightStyle?.paddingBottom}
-          paddingRight={step.spotlightStyle?.paddingRight}
-          paddingLeft={step.spotlightStyle?.paddingLeft}
-        />
-      ) : (
-        <ModalOverlay />
-      )}
+      <TourOverlay
```
Comment (microbit-matt-hillsdon) [inline src/pages/Tour.tsx:86]: I think I did this for a good reason. Will try to dredge out why!

Comment (microbit-robert) [reply src/pages/Tour.tsx:86]: Yes, I am worried I'm missing something.

Comment (microbit-matt-hillsdon) [reply src/pages/Tour.tsx:86]: It's the MakeCode tour which has a modal on a modal.
Couldn't find an easy way to make that work with the TourOverlay.
A cheat option is to only use ModalOverlay for single step tours with no selector. That just happens to work for us for now until perhaps we sort the modal nature of MakeCode.

Comment (microbit-matt-hillsdon) [reply src/pages/Tour.tsx:86]: Hack version here: https://github.com/microbit-foundation/ml-trainer/pull/458

**Comment by microbit-matt-hillsdon** [comment]
Closing in favour of https://github.com/microbit-foundation/ml-trainer/pull/458

**Pull request closed without merging**